IHP/View/CSSFramework.hs

```diff
@@ -433,7 +433,7 @@ tailwind = def
                     placeholder={placeholder}
                     id={fieldInputId}
                     class={classes [inputClass, (inputInvalidClass, isJust validatorResult), (fieldClass, not (null fieldClass))]}
-                    value={fieldValue}
+                    value={maybeValue}
```

Comment:
@mpscholten I tried the Maybe value added in https://github.com/digitallyinduced/ihp/commit/8c5b9cbe5a360542b9fc95ea4032401b17440203

But it still seems to output the `value=""` attribute. Anything obvious I'm missing? :smile_cat: